src/server/network/protocol/protocolgame.cpp

```diff
@@ -7738,12 +7734,6 @@ void ProtocolGame::AddCreature(NetworkMessage &msg, const std::shared_ptr<Creatu
 	if (!creature->isInGhostMode() && !creature->isInvisible()) {
 		const Outfit_t &outfit = creature->getCurrentOutfit();
 		AddOutfit(msg, outfit);
-		if (!oldProtocol && outfit.lookMount != 0) {
-			msg.addByte(outfit.lookMountHead);
-			msg.addByte(outfit.lookMountBody);
-			msg.addByte(outfit.lookMountLegs);
-			msg.addByte(outfit.lookMountFeet);
-		}
 	} else {
```

Comment:
load this (sendCreatureOutfit and AddCreature) into AddOutfit
 (same packets)
![image](https://github.com/user-attachments/assets/1f96b17a-0a03-404e-bd24-2597f2a6e5e0)